Remove old docs and update mkdocs and md files to reflect the new ver… (#83)
Harvard-Neutrino/prometheus: docs/usage.md

# Usage

> ## Running simulations

Reviewer: I am sorry, just looked through the site and all headings are capitalized, let's revert this and capitalize them, sorry 😭  The style guide is very much in order.

I'll submit committable suggestions so that you don't have to do this manually (you can click "add to batch" and commit all of them in one go).

```suggestion
## Running Simulations
```

The [examples directory](https://github.com/Harvard-Neutrino/prometheus/tree/main/examples) of the Prometheus GitHub repository contains small, runnable scripts that demonstrate typical Prometheus workflows.

Recommended quick-start examples:

- `examples/01_basic_water.py` — Minimal water-case example to validate an install.
- `examples/02_basic_ice.py` — Minimal ice-case example (uses ppc) to validate an install with ppc.

To execute the examples, you need to activate the repository-local micromamba environment.

Run:

```sh
source scripts/activate.sh .prometheus_env
```

Then run an example script:

```sh
python examples/<SCRIPT_NAME>
```

Replace `<SCRIPT_NAME>` with the name of your script: `01_basic_water.py` or `02_basic_ice.py` for a quick-start option.

> ## Getting help

Reviewer: ```suggestion
## Getting Help
```

## Running Simulations

> The [examples directory](https://github.com/Harvard-Neutrino/prometheus/tree/main/examples) contains small, runnable scripts that demonstrate typical Prometheus workflows.

Reviewer: I see why omitting the mentions of the repo makes sense (cause users would arrive to doc site from the repo), but just to be more explicit for the beginners' sake, I would keep it here mentioned once at the beginning.

```suggestion
The [examples directory](https://github.com/Harvard-Neutrino/prometheus/tree/main/examples) of the Prometheus GitHub repository contains small, runnable scripts that demonstrate typical Prometheus workflows.
```

Recommended quick-start examples:

- `examples/01_basic_water.py` — Minimal water-case example to validate an install.
- `examples/02_basic_ice.py` — Minimal ice-case example (uses ppc) to validate an install with ppc.

> Run examples using the repository-local micromamba environment. First, activate it:

Reviewer: ```suggestion
To execute the examples, you need to activate the repository-local micromamba environment.

Run: 
```

```sh
source scripts/activate.sh .prometheus_env
```

Then run a water simulation:

```sh
python examples/01_basic_water.py
```

Or an ice simulation:

```sh
python examples/02_basic_ice.py
```

You can also combine activation and execution on one line:

```sh
source scripts/activate.sh .prometheus_env && python examples/01_basic_water.py

> ```

Reviewer: Can't use the suggestion block, so pasting a suggestion as text below.
Reasoning: we want to be explicit for the beginners, but also we want to respect users' time and attention space. This passage conveys the same meaning without extra noise.

***

Then run an example script:

```sh
python examples/<SCRIPT_NAME>
```

Replace `<SCRIPT_NAME>` with the name of your script: `01_basic_water.py` or `02_basic_ice.py` for a quick-start option.
